packages/appstore/src/components/wallet-content/__tests__/wallet-transfer-block.spec.tsx

```diff
@@ -6,15 +6,21 @@ import { TWalletAccount } from 'Types';
 import { StoreProvider, mockStore } from '@deriv/stores';
 
 const wallet_account: TWalletAccount = {
-    name: 'USD',
     currency: 'USD',
     icon: '',
     balance: 10415.24,
-    icon_type: 'fiat',
     landing_company_name: 'svg',
-    is_disabled: false,
-    is_virtual: false,
+    is_disabled: 0,
+    is_virtual: 0,
     loginid: 'CRW10001',
+    is_selected: false,
+    is_demo: false,
+    wallet_currency_type: '',
+    is_malta_wallet: false,
+    gradient_header_class: 'wallet-header__test',
+    gradient_card_class: 'wallet-card__test',
+    currency_config: undefined,
+    linked_to: [{ loginid: 'CR1001', platform: 'dtrade' }],
```

Comment:
```suggestion
    linked_to: [{ loginid: 'CR1001', platform: 'trading' }],
```

we also need account's balance here

Reply:
<img width="613" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/833ea5a5-a14a-48b0-b10b-51f2f041e6cf">

I suppose we don't have 'trading' in platform. We have `trading` only in `account_category`